Framework/Algorithms/src/MagFormFactorCorrection.cpp

```diff
@@ -101,7 +102,7 @@ void MagFormFactorCorrection::exec() {
     auto &Y = outputWS->mutableY(i);
     auto &E = outputWS->mutableE(i);
     for (int64_t j = 0; j < specSize; j++) {
-      double ff = (numAxes == 1) ? FF[j] : FF[i];
+      double ff = (iax == 0) ? FF[j] : FF[i];
```

Comment:
Need `iax` here (line 105) because it's used as the index of the Q axis (which could be x or y). It's not the same as `numAxes-1` because of the `break` in line 74.